lib/__tests__/node.test.ts

```diff
@@ -0,0 +1,15 @@
+import insightsClient from "../node";
+
+describe("node", () => {
+  it("correctly return userToken", done => {
```

Comment:
We can write two different test cases for the two APIs to maintain them more easily.